__tests__/integration.test.js

```diff
@@ -153,6 +153,78 @@ describe('POST/api/articles/:article_id/comments', () => {
     });
 });
 
+describe('PATCH: api/articles/:article_id', () => {
+    test('200: identifies article id and adds to an articles votes value when given integer', () => {
+        const articleVotes = {
+            inc_votes: 8
+        };
+        return request(app)
+        .patch('/api/articles/3')
+        .send(articleVotes)
+        .expect(201)
+        .then(({ body }) => {
+            expect(body.votes).toBe(8)
+            expect(body).toHaveProperty("author");
+            expect(body).toHaveProperty("title");
+            expect(body).toHaveProperty("article_id");
+            expect(body).toHaveProperty("topic");
+            expect(body).toHaveProperty("created_at");
+            expect(body).toHaveProperty("votes");
+            expect(body).toHaveProperty("article_img_url");
+        });
+    });
```

Comment:
Really good first test, but there seems to be a conflict between your test expected status code and the actually status code you are testing for here. The more accurate status code here would be 200 as 201 usually means something new has been added, but in this case we are just updating something so 200 is what I would recommend.